File: C++ for Everyone, 2nd Ed, Horstmann/Chapter4/P4-13.cpp
//Write a program that reads a word and prints the number of vowels in the word.For
//this exercise, assume that  a e i o u y are vowels.For example, if the user pro vides the
//input  "Harry", the program prints  2 vowels

#include <iostream>
#include <string>

using namespace std;

int main()
{
   int count = 0;

   cout << "Enter a word: ";
   string word;
   cin >> word;

   for(int i = 0; i < word.length(); i++)
   {
      string letter = word.substr(i, 1);

      if(letter == "a" || letter == "e" ||
          letter == "i" || letter == "o" ||
          letter == "u" || letter == "y" ||
          letter == "A" || letter == "E" ||
          letter == "I" || letter == "O" ||
          letter == "U" || letter == "Y")
      {
         count++;
      }
   }

   cout << "There were " << count << " vowels." << endl;

   return 0;
}

File: Python-For-Everyone-Horstmann/Chapter4-Loops/P4.13.py
# Write a program that reads an integer value and prints all of its binary digits in
# reverse order: Print the remainder  number % 2 , then replace the number with  number //
# 2 . Keep going until the number is 0. For example, if the user provides the input 13,
# the output should be
#     1
#     0
#     1
#     1

inputN = int(input("Enter a number: "))

while inputN > 0:
    print(inputN % 2)
    inputN = inputN // 2